### DISCUSSION TITLE
Function that takes no arguments

### DISCUSSION THREAD

**Discussion by synthe102** [mlua-rs/mlua#253] [Q&A] [answered]
Hi !

I'd like to implement a function that takes no arguments.
However, `Lua.create_function()` expects the rust function to take two arguments.
How can I solve this ?

Thanks !

**Comment by khvzak** [comment, marked as answer]
```rust
let func = lua.create_function(|_, ()| Ok("hello")).unwrap();
```

**Comment by alerque** [comment]
Closely related, but the other way around I have a question at #266 about how to *call* a function without passing any args.
